bin/docker-build-bb.sh

```diff
@@ -1,9 +1,2 @@
 #!/usr/bin/env bash
-
-set -e
-target_dir=target
-
-rm -rf ${target_dir}
-mkdir ${target_dir}
-GOOS=linux go build -o target/bb .
 docker build  -f ./Dockerfile . -t buoyantio/bb:v0.0.1
```

Comment:
Total TIOLI suggestion -- it might make sense to use the "latest" tag here, instead of "v0.0.1". Otherwise we'll need to remember to update it with each release.